crates/ya-http-proxy-model/src/addr.rs

```diff
@@ -72,3 +72,18 @@ impl Display for Addresses {
         write!(f, "]")
     }
 }
+
+impl Addresses {
+    pub fn gather_and_sort_addresses(
```

Comment:
this is a very bad coding style.

1. Why associated function?
` addr::Addresss:: gather_and_sort_addresses(...)` looks better than simple `addr::gather_and_sort_addresses`. 

2. Why this is a public API for address?

this is a very specific name `gather` and `sort`. it does not look like something general to be done on adresses.